.travis.yml

```diff
@@ -54,4 +55,7 @@ after_success:
     - test "${use_codecov}" == "true" && codecov || true
 
 after_failure:
+    - sudo cp -av /var/lib/systemd/coredump exported-artifacts/
```

Comment:
Why do we have to collect the coredump from the host and not from inside the container?
https://github.com/nmstate/nmstate/blob/master/automation/run-tests.sh#L145 was meant to collect coredumps from the container and not from the host. In the past I saw core dumps in the container and not on the host IIRC.

Reply:
When segfault happens in container, only the host can save the coredump.

Reply:
No idea why and not interested. Might be some systemd evils again.